_docs/quickstarts/scanner/index.md

```diff
@@ -28,9 +28,9 @@ To run Forseti Scanner, follow the process below:
 
   1. Run the scanners:
 
-          ```bash
-          $ forseti_scanner --forseti_config <path to forseti_conf.yaml>
-          ```
+  ```bash
+  $ forseti_scanner --forseti_config <path to forseti_conf.yaml>
```

Comment:
Should we remove the "$ " preceding the command so users can just click the "copy" icon and paste into their terminal?